pos-print/src/main/java/com/clouway/pos/print/core/InMemoryPrintQueue.kt

```diff
@@ -0,0 +1,29 @@
+package com.clouway.pos.print.core
+
+import org.eclipse.jetty.util.BlockingArrayQueue
+
+/**
+ * @author Tsvetozar Bonev (tsvetozar.bonev@clouway.com)
+ */
+class InMemoryPrintQueue : PrintQueue {
+
+  private val queue = BlockingArrayQueue<PrintReceiptRequest>()
```

Comment:
ArrayBlockingQueue is java equivalent which is better to be used as this one is related to jetty.

Reply:
My mistake, changed to java blocking queue.